widget/ui/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,77 @@
+import type { PropTypes } from './Tabs.types';
+
+import React, { useEffect, useRef, useState } from 'react';
+
+import { Divider } from '../Divider';
+import { Tooltip } from '../Tooltip';
+
+import { BackdropTab, Tab, Tabs } from './Tabs.styles';
+
+export function TabsComponent(props: PropTypes) {
+  const {
+    items,
+    onChange,
+    container = document.body,
+    value,
+    currentTabIndex,
```

Comment:
I think this component does not need to receive "currentTabIndex" as a prop. It already has the value prop and you can find the current TabIndex by checking id of each item. Also I think you can improve the finding process using "useMemo" hook.

Reply:
fixed